davinci-grid.md

```diff
@@ -30,24 +30,38 @@ to some special commands provided by `ganga`.
 To create your first `ganga` job type the following:
 
 ```python
-j = Job(application=DaVinci(version='v41r2'))
+j = Job(name='First ganga job')
+myApp = prepareGaudiExec('DaVinci','v41r2', myPath='.')
+j.application = myApp
```

Comment:
slightly off topic, but IMHO it's worth stressing that `prepareGaudiExec` has observable side effects (does `lb-dev`). Having it on a separate line (even moving it before `Job()` helps with that.